piControlMain.c

```diff
@@ -274,20 +279,31 @@ static int __init piControlInit(void)
 
 	if ((piDev_g.machine_type == REVPI_CONNECT) ||
 	    (piDev_g.machine_type == REVPI_CONNECT_SE) ||
+	    (piDev_g.machine_type == REVPI_CONNECT_4) ||
 	    (piDev_g.machine_type == REVPI_FLAT)) {
 		res = res
 		   || devm_led_trigger_register(piDev_g.dev, &piDev_g.a3_green)
 		   || devm_led_trigger_register(piDev_g.dev, &piDev_g.a3_red);
 	}
 
-	if (piDev_g.machine_type == REVPI_FLAT) {
+	if ((piDev_g.machine_type == REVPI_FLAT) ||
+	    (piDev_g.machine_type == REVPI_CONNECT_4)) {
```

Comment:
You don't need these double parentheses because `==` has higher <a href="https://www.swansontec.com/sopc.html">precedence</a> than `||`.

So please change to:

```
	if (piDev_g.machine_type == REVPI_FLAT ||
	    piDev_g.machine_type == REVPI_CONNECT_4) {
```

Yes I can see that's it's also done this way above, but that doesn't make it better. :)